src/Microsoft.OData.ModelBuilder/Microsoft.OData.ModelBuilder.xml

```diff
@@ -5106,5 +5116,2153 @@
             <param name="isExpandable">This entity set supports the expand expressions.</param>
             <param name="nonExpandableProperties">These properties cannot be used in $expand expressions.</param>
         </member>
+        <member name="T:Microsoft.OData.ModelBuilder.Capabilities.V1.AcceptableEncodingsConfiguration">
+            <summary>
+            List of acceptable compression methods for ($batch) requests, e.g. gzip
```

Comment:
@g2mula If this docstring is valid in this context then its very hard to relate it to `AcceptableEncodings`

Reply:
Unfortunately this is the documentation for the EdmTerm as present in the referenced model.